go.mod

```diff
@@ -8,15 +8,14 @@ require (
 	github.com/go-logr/logr v0.1.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
-	github.com/talos-systems/crypto v0.3.1
-	github.com/talos-systems/talos/pkg/machinery v0.11.3
-	golang.org/x/sys v0.0.0-20210816074244-15123e1e1f71
+	github.com/talos-systems/crypto v0.3.2
+	github.com/talos-systems/talos/pkg/machinery v0.12.3-0.20210920195258-7e63e43eb399
```

Comment:
0.12.2 + secrets bundle fix, basically can be anything like 0.12.3 once we actually release that